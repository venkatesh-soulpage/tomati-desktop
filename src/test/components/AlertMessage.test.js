import React from "react";
import { render } from "@testing-library/react";
import AlertMessage from "components/AlertMessage";
import { Router } from "react-router-dom";
import history from "utils/history";
import { Provider } from "react-redux";
import configureStore from "store";

describe("AlertMessage page", () => {
  it("matches snapshot", () => {
    const store = configureStore();
    const { asFragment } = render(
      <Provider store={store}>
        <Router history={history}>
          <AlertMessage />
        </Router>
      </Provider>,
      {}
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
