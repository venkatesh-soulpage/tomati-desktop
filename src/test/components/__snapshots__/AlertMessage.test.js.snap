// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertMessage page matches snapshot 1`] = `<DocumentFragment />`;
